vimeo-networking/src/main/java/com/vimeo/networking/utils/VimeoNetworkUtil.java

```diff
@@ -199,8 +200,7 @@ public void run() {
                         .responseBodyConverter(VimeoError.class, new Annotation[0]);
                 vimeoError = errorConverter.convert(response.errorBody());
             } catch (final Exception e) {
-                //noinspection CallToPrintStackTrace
-                e.printStackTrace();
+                ClientLogger.e("Error while attempting to convert response body to VimeoError", e);
```

Comment:
The methods `VimeoNetworkUtils.getSimpleQueryMap` and `VimeoNetworkUtils.getErrorFromResponse` could be updated to use the ClientLogger when an exception is caught.

Reply:
That's what I did in this commit: https://github.com/vimeo/vimeo-networking-java/pull/391/commits/497dfb8e6c193957aff488e477c3939379648e98, or are you referring to something else.

Reply:
oh. Sorry.  I got mixed with something else.